examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def get_batches(X,y):
</pre><p>NIT: Leave one empty space between "," and "y"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Resolved.